src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -16,18 +17,38 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
 
     init() {
         this.selection = [];
+        this.showChecked();
         this.counter = DomAccess.querySelector(this.el, this.options.countSelector);
         this._registerEvents();
+
     }
 
     /**
      * @private
      */
     _registerEvents() {
         const checkboxes = DomAccess.querySelectorAll(this.el, this.options.checkboxSelector);
-
+        const arrowIcon = DomAccess.querySelectorAll(this.el,this.options.arrowIcon);
+        const current = this; // because some functions need to call inside
+                              // another object, for example assigning an event
+
+        if(arrowIcon !== false)
+        {
```

Comment:
In general, we style the JS with the opening bracket on the same line. There is already a story under Review which would introduce JS linting which would resolve most of these styling issues, but for the time being, please ensure the formatting is similar to the existing code base. 

Make the changes in other areas of the file as well where this is applicable.